docs/API.md

```diff
@@ -69,32 +98,83 @@ gulp.src('./client/templates/*.jade')
 ```
 
 The write path is calculated by appending the file relative path to the given
-destination directory. In turn, relative paths are calculated against the file base. 
-See `gulp.src` above for more info.
+destination directory. In turn, relative paths are calculated against 
+the file base. See `gulp.src` above for more info.
 
 #### path
 Type: `String` or `Function`
 
-The path (output folder) to write files to. Or a function that returns it, the function will be provided a [vinyl File instance](https://github.com/wearefractal/vinyl).
+The path (output folder) to write files to. Or a function that returns it,
+the function will be provided a [vinyl File instance].
+
+#### options
+Type: `Object`
+
+##### options.cwd
+Type: `String`
+
+Default: `process.cwd()`
+
+`cwd` for the output folder, only has an effect if provided output folder is 
+relative.
+
+##### options.mode
+Type: `String`
+
+Default: the mode of the input file (file.stat.mode) or the process mode 
+if the input file has no mode property.
+
+Octal permission string specifying the mode the files should be created with. 
```

Comment:
@phated Should I reword it to:

##### options.mode
Type: `String` or `Number`

Default: the mode of the input file (file.stat.mode) or the process mode if the input file has no mode property.

Octal permission specifying the mode the files should be created with: e.g. `"0777"`, `0777` or `511` (`0777` in base 10).